lms/djangoapps/grades/models.py

```diff
@@ -362,7 +375,7 @@ def update_or_create_grade(cls, **params):
         Wrapper for objects.update_or_create.
         """
         cls._prepare_params(params)
-        VisibleBlocks.cached_get_or_create(params['visible_blocks'])
+        VisibleBlocks.cached_get_or_create(params['user_id'], params['visible_blocks'])
```

Comment:
Nit: `user_id` is popped from `params` below as well.  Does it read better if that line of code is moved up and DRYed?